source/fundamentals/authentication/mechanisms.txt

```diff
@@ -233,28 +233,31 @@ The driver checks for your credentials in the following sources in order:
 
 The ``X.509`` authentication mechanism uses
 :wikipedia:`TLS <Transport_Layer_Security>` with X.509 certificates to
-authenticate your user, identified by the distinguished name (DN) of your
-client certificate. For more information on determining the *subject* name from
-the X.509 certificate, see the MongoDB manual :manual:`X.509 tutorial
-</tutorial/configure-x509-client-authentication/#add-x-509-certificate-subject-as-a-user>`.
+authenticate by retrieving the distinguished name (DN) from the
+client certificate.
```

Comment:
Question:
I thought the removed sentence provides information that helps the user determine the certificate is the one that corresponds to their MongoDB user? Is there enough info in this section on how to set up x.509 authentication without it?

Reply:
It was my understanding that since username is no longer required, users do not need to know how to retrieve the subject from their certificate.

Reply:
Ok. It wasn't clear to me whether the link was trying to confirm the certificate was for the user or to extract the subject/username.